packages/lesswrong/components/posts/PostsItemMeta.jsx

```diff
@@ -24,13 +29,13 @@ const PostsItemMeta = ({classes, currentUser, post, read, timezone}) => {
       </MetaInfo>}
 
       { post.isEvent && <MetaInfo>
-        <Tooltip title={
-          post.startTime ? <EventTime post={post} /> : <span>To Be Determined</span>}
-          >
-          {post.startTime ? <span>{moment(post.startTime).tz(timezone).format("MMM Do")}</span>
-            : <span>TBD</span>
-          }
-        </Tooltip>
+        {post.startTime
+          ? <Tooltip title={<EventTime post={post} />}>
+              <DateWithoutTime date={post.startTime} />
```

Comment:
Possibly also here? Or, if FormatDate doesn't currently do quite the thing (I'm not sure I grok what's going on with Timezones, update it so it does? I think our date needs are pretty simple and don't need multiple components)

Reply:
FormatDate doesn't do the thing because it comes with its own tooltip, and this tooltip isn't something FormatDate knows how to handle: EventDate is potentially *two* dates, ie a start time and end time.